canonical-web-and-design/juju.is
# Update and Switch over to github actions

## Done

Replace Circle CI with Github actions

## QA

Github actions checks should turn green:
run-image
run-dotrun
lint-scss
lint-python
test-python

## Issue / Card

Testing sprint
